api/hypershift/v1beta1/azure.go

```diff
@@ -284,12 +290,13 @@ type AzureNodePoolOSDisk struct {
 	SizeGiB int32 `json:"sizeGiB,omitempty"`
 
 	// storageAccountType is the disk storage account type to use.
-	// Valid values are Standard, StandardSSD, PremiumSSD and UltraSSD and omitted.
+	// Valid values are Premium_LRS, PremiumV2_LRS, Standard_LRS, StandardSSD_LRS, UltraSSD_LRS.
```

Comment:
since we are here, can you update the godoc r/storageAccountType/diskStorageAccountType